Plugins/NativeEngine/Source/NativeEngine.cpp

```diff
@@ -1061,7 +1061,7 @@ namespace Babylon
 
         assert(matrix.size() == size * size);
 
-        if constexpr (size < 4)
+        if constexpr (size < 3)
         {
             std::array<float, 16> matrixValues{};
```

Comment:
Do we have to worry about any sort of side effects by storing a 3x3 matrix across a size 16 array?  I'm assuming that the untouched elements of the array start as 0 but I wanted to make sure.

Reply:
The `{}` will initialize the array to the default value of `float` which is `0.0f`

Reply:
Disregard, I misread this.